.gitignore

```diff
@@ -10,6 +10,7 @@ coverage
 dist
 node_modules
 tsconfig*.tsbuildinfo
+~

```

Comment:
curious what this is for

Reply:
Related to pnpm files getting loaded locally - didn't want to push them to remote.